tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -44,6 +71,10 @@ TEST( SetOnOffMassFlowRateTest, Test1 )
 	MSHPMassFlowRateHigh = 0.0;
 
 	UnitarySystem.allocate( 1 );
+	UnitarySystem( UnitarySysNum ).SysAvailSchedPtr = GetScheduleIndex( "FanAndCoilAvailSched" ); // "Get" the schedule inputs
+	UnitarySystem( UnitarySysNum ).FanAvailSchedPtr = GetScheduleIndex( "FanAndCoilAvailSched" );
+	Schedule( 1 ).CurrentValue = 1.0; // set availability and fan schedule to 1
+
```

Comment:
Not sure how this unit test passed before. The schedules were 0 so air flow should have also been 0. Anyway, this test now passes.